samples/sample_vpp/src/sample_vpp_parser.cpp

```diff
@@ -252,6 +252,11 @@ msdk_printf(MSDK_STRING("   [-mirror (mode)]      - mirror image using specified
 
 msdk_printf(MSDK_STRING("   [-n frames] - number of frames to VPP process\n\n"));
 
+#if MFX_VERSION >= MFX_VERSION_NEXT
+msdk_printf(MSDK_STRING("   [-cf_red R]           - enable colorfill (red component).\n"));
```

Comment:
That's an overkill for the user. Let's have a simple option:
`[-cf #RRGGBB] - use specified color for the colorfill (in 8-bit rgb format)`

Also, you miss a code which will convert the colorfill RGB value to YUV color space - copy it from the library if it is currently missed in the sample.